worker-message-prioritization-rerouting/src/main/java/com/github/workerframework/workermessageprioritization/rerouting/StagingQueueCreator.java

```diff
@@ -105,7 +107,12 @@ void createStagingQueue(final Queue targetQueue, final String stagingQueueName)
         final boolean durable = targetQueue.isDurable();
         final boolean exclusive = targetQueue.isExclusive();
         final boolean autoDelete = targetQueue.isAuto_delete();
-        final Map<String, Object> arguments = targetQueue.getArguments();
+        Map<String, Object> arguments = new HashMap<>();
```

Comment:
In quorum mode the following means we'll ignore all arguments from the target queue, when we only need to ignore the queue type and max priority.

On reflection we only did this because we hoped to run one work in quorum mode and later discovered this was too problematic, in summary I don't think we need this change.

Reply:
By this do you mean we don't need any of this change? Would we not still need to put the quorum headers onto the arguments to ensure the staging queues are created as quorum queues if the environment variable is set?

Reply:
The staging queue is created using a copy of the arguments applied to the target queue, if everything is using quorum queues then the staging queue will be created with the same arguments as the target.

targetQueue.getArguments();